backend/internal/db/health_value.go

```diff
@@ -13,19 +14,12 @@ type RepoHealthMetrics struct {
 	Id                  uuid.UUID `json:"id"`
 	RepoId              uuid.UUID `json:"repoid"`
 	CreatedAt           time.Time `json:"createdat"`
-	ContributerCount    int       `json:"contributercount"`
+	ContributorCount    int       `json:"contributorcount"`
 	CommitCount         int       `json:"commitcount"`
 	SponsorCount        int       `json:"sponsorcount"`
 	RepoStarCount       int       `json:"repostarcount"`
 	RepoMultiplierCount int       `json:"repomultipliercount"`
-	RepoWeight          float64   `json:"reposponsordonated"`
-}
-
-type InternalHealthMetrics struct {
-	SponsorCount        int     `json:"sponsorcount"`
-	RepoStarCount       int     `json:"repostarcount"`
-	RepoMultiplierCount int     `json:"repomultipliercount"`
-	RepoWeight          float64 `json:"reposponsordonated"`
+	RepoWeight          int       `json:"activeffsusercount"`
```

Comment:
Still RepoWeight as name? Changed in other files, should this also be changed?

Reply:
these items are refactored in another branch. Once ffs108 is merged into sa branch and you merge this branch it should be fine.